### PR TITLE
Add missing EDD timeline operations

### PR DESCRIPTION
**Pull request by ashkarpetin**
## Summary
Add missing EDD timeline operations
Fix EDD timeline operation IDs
Fix Plaid credential hashes operation ID

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+    '404':
+      $ref: ../components/responses/NotFound.yaml
+delete:
+  x-products:
+    - Core
+  tags:
+    - Customers timeline
+  summary: Delete EDD timeline message
```
Comment (ashkarpetin) [inline openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml:36]:
```suggestion
  summary: Delete an EDD timeline message
```

File: openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+  - name: messageId
+    in: path
+    description: EDD timeline message ID.
```
Comment (noon-dawg) [inline openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml:5]: nitpick: based on the style
```suggestion
    description: ID of the EDD timeline message.
```

Comment (ashkarpetin) [reply openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml:5]: I copy pasted from https://github.com/Rebilly/api-definitions/blob/4cdf5b21c53ee08f6d58c7abf993cdde0ded9046/openapi/paths/customers%40%7Bid%7D%40timeline%40%7BmessageId%7D.yaml#L5

Comment (noon-dawg) [reply openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml]: It seems we still have some inconsistencies on this. I'll make a PR

**Comment by ashkarpetin** [comment]
php please review

**Comment by ashkarpetin** [comment]
js please review

File: openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    required: true
+    schema:
+      type: string
+get:
+  x-products:
+    - Core
+  tags:
+    - Customers timeline
+  summary: Retrieve an EDD timeline message
+  operationId: GetCustomerEddTimeline
```
Comment (subenson) [inline openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml:15]:
```suggestion
  operationId: GetCustomerEddTimelineMessage
```

File: openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      $ref: ../components/responses/Forbidden.yaml
+    '404':
+      $ref: ../components/responses/NotFound.yaml
+delete:
+  x-products:
+    - Core
+  tags:
+    - Customers timeline
+  summary: Delete an EDD timeline message
+  operationId: DeleteCustomerEddTimeline
```
Comment (subenson) [inline openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml:37]: ?
```suggestion
  operationId: DeleteCustomerEddTimelineMessage
```

File: openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml (hunk 3 above)
Comment (ashkarpetin) [reply openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml:15]: No

File: openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml (hunk 4 above)
Comment (ashkarpetin) [reply openapi/paths/customers@{id}@edd-timeline@{messageId}.yaml:37]: No, check internal discussion

**Comment by ashkarpetin** [comment]
php please review

**Comment by ashkarpetin** [comment]
js please review

**Comment by ashkarpetin** [comment]
php please review

**Comment by ashkarpetin** [comment]
js please review

**Review by dara123 — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Pull request merged**